src/main/java/com/teragrep/cfe18/handlers/entities/CaptureFile.java

```diff
@@ -52,7 +52,7 @@
 public class CaptureFile {
 
     public enum CaptureType {
```

Comment:
enums should be in their relevant file CaptureFile.java and CaptureRelp.java include the same enum, using one should be enough?